doc/nrf/config_and_build/bootloaders_and_dfu/fw_update.rst

```diff
@@ -329,18 +352,63 @@ If this image has, in order of precedence, a *major*, *minor*, or *revision* val
 .. note::
 
    The optional label or build number specified after the ``+`` character is ignored when evaluating the version.
-   An existing application image with version ``0.1.2+3`` can be overwritten by an uploaded image with ``0.1.2+2``, but not by one with ``0.1.1+2``.
+   For example, an existing application image with version ``0.1.2+3`` can be overwritten by an uploaded image with ``0.1.2+2``, but not by one with ``0.1.1+2``.
 
-Hardware-based downgrade prevention
------------------------------------
+.. _ug_fw_update_downgrade_protection_hw:
+.. _bootloader_monotonic_counter:
 
-Hardware-based downgrade prevention is implemented with a counter value stored in the *provision* partition of the non-volatile storage.
+Hardware-based downgrade protection
+===================================
 
-See the :ref:`bootloader_monotonic_counter` section for how to enable and configure this feature.
+.. bootloader_monotonic_counter_start
+
+You can implement hardware-based downgrade protection using a non-volatile monotonic counter.
+
+Counter updates are written to slots in the *Provision* area, with each new counter update occupying a new slot.
+For this reason, the number of counter updates, and therefore firmware version updates, is limited.
+The *Provision* is a partition in non-volatile memory, and its location can be found using :ref:`pm_generated_output_and_usage_pm_report`.
+
+Using a counter is optional and can be configured for the application using configuration options.
+You can also configure the supported number of updates, but the number is limited by the size of the *Provision* area and how much of that area is taken up by other features, like public key hashes.
+In addition, you can configure what firmware version of the image you want to boot.
+
+.. bootloader_monotonic_counter_end
+
+.. _ug_fw_update_hw_downgrade_nsib:
+
+Downgrade protection using |NSIB|
+=================================
+
+.. bootloader_monotonic_counter_nsib_start
+
+To enable anti-rollback protection with monotonic counter for |NSIB|, set the following configurations in the application (parent image):
```

Comment:
I am not sure if this does not need to be set for both bootloader and application. @sigvartmh, could you confirm, please?

Reply:
For this setting it is enough to just set it in the application(parent image).

Reply:
Thanks for the clarification. In that case I think we are ready to merge this one and further improvements can be added in separate PRs.